### ISSUE TITLE
Color palette find / replace

### ISSUE THREAD

**Issue by samanthafoust** [wp-cli/ideas#110]
Hello,

I'd love to see a command that brings up all colors used in CSS on a site, ideally with those colors displayed in the terminal next to the value or the value itself be that color. This would allow people to easily use search / replace to unify inconsistent palettes or completely change color palettes.

A bulk remove of all inline styling would be nice to for developers who are taking over poorly made websites, if people like this should I make another ticket?

~ Samantha

**Comment by joyously** [comment]
Are you referring only to colors (not other styles), and only in post content? 
There can be widgets with styles (which are saved in options table). There can also be styles in the theme and in plugins, and soon with Gutenberg, in core. Also, there can be styles using custom properties (CSS variables) that are colors but don't look like colors.

**Comment by swissspidy** [comment]
If you want to really know which colors are used on a website, the only way to do this is to run a browser that visits, parses, and analyzes the site.

There are way better CLI tools and web apps that can do this than WP-CLI.

Removing inline styling from your site is a whole other topic for itself. What if a plugin you use adds inline CSS? You can't just remove this CSS. Not with WP-CLI anyway. For this you'd use some site optimization plugin — or just install a different plugin.

Long story short: I don't think this could work, nor is WP-CLI the right tool for it.
